package.json

```diff
@@ -98,15 +103,6 @@
       ],
       [
         "@pika/plugin-build-web"
-      ],
-      [
-        "@pika/plugin-bundle-web",
-        {
-          "browser": true
-        }
-      ],
-      [
-        "pika-plugin-unpkg-field"
```

Comment:
Should probably also remove them from devDeps since they are no longer used

Reply:
Ah yes very good catch!